<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonDialog</class>
 <widget class="QDialog" name="ButtonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>Output Target</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="commandLabel">
       <property name="font">
        <font>
         <family>Bitstream Charter</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameText"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <family>Bitstream Charter</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="commandText"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="outputText">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ButtonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
